WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingNavigationController.swift

```diff
@@ -5,25 +5,12 @@ protocol PrepublishingDismissible {
 }
 
 class PrepublishingNavigationController: LightNavigationController {
-    override func viewDidLoad() {
-        super.viewDidLoad()
-
-        // Set the height for iPad
-        if UIDevice.isPad() {
-            preferredContentSize = Constants.iPadPreferredContentSize
-        }
-    }
-
     // We are using intrinsicHeight as the view's collapsedHeight which is calculated from the preferredContentSize.
     override public var preferredContentSize: CGSize {
         set {
             viewControllers.last?.preferredContentSize = newValue
```

Comment:
Would it be good to also call `super.preferredContentSize = newValue` in order to [trigger popovers to automatically update their size](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621476-preferredcontentsize) if needed? 
From the docs:
> Changing the value of this property while the view controller is being displayed in a popover animates the size change

Reply:
Works for me, done in: f9835642f3068460ee6b7f29f11737e65014bcdc